Healthy/Classes/Modules/Dashboard/Views/HomeHeaderView.swift

```diff
@@ -1,12 +1,55 @@
 import UIKit
 
 final class HomeHeaderView: UIView {
+    
+    // MARK: Outlets
+    
+    @IBOutlet private(set) weak var contentView: UIView!
+    @IBOutlet private(set) weak var titleLabel: UILabel!
+    @IBOutlet private(set) weak var subtitleLabel: UILabel!
+    @IBOutlet private(set) weak var userImageView: UIImageView!
+    
+    // MARK: Init
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        initView()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        initView()
+    }
+
+    // MARK: Configurations
+
+    private func initView() {
+        loadViewFromNib()
+        configureLayout()
+    }
+    
+    private func configureLayout() {
+        titleLabel.applyStyle(.signupTitle)
+        
+        // TODO: - Waiting for add home subtitle label style.
+        subtitleLabel.applyStyle(.signupSubtitle)
+        
+        NSLayoutConstraint.activate([
+            contentView.heightAnchor.constraint(equalToConstant: 128),
+            contentView.widthAnchor.constraint(equalToConstant: UIScreen.main.bounds.width)
+        ])
```

Comment:
```suggestion
```

Not needed, the width will be inferred from the parent view. The height should fit the content.

Reply:
okay, but how make height of view fit the content if we not assigned. 🤔